docs/develop/requirements-property-unified-manifest.md

```diff
@@ -0,0 +1,203 @@
+
+### extensions.alternates.requirements
+
+The "extensions.alternates" property enables add-in developers to do the following:
+
+- Maintain a version of an add-in that was built on an older extensibility platform (such as COM or VSTO add-ins) or using the XML manifest, in addition to the version that uses the unified manifest.
+- Either hide or give preference to the version that uses the older technology.
+- Specify icons that are needed to make the unified manifest version of the add-in installable on Office versions that don't directly support the unified manifest. 
+
+[!INCLUDE [non-unified manifest clients note](../includes/non-unified-manifest-clients.md)]
+
+For more information, see [Manage both a unified manifest and an XML manifest version of your Office Add-in](../concepts/duplicate-legacy-metaos-add-ins.md).
+
+The "requirements" subproperty of "extensions.alternates" to selectively apply the "hide" or "prefer" subproperties only when certain requirements are met. 
+
+For example, suppose that you want to hide (from the Office UI for installing add-ins) an older version of your add-in, but only in Office versions that support the **Mailbox 1.10** requirement set. You could do that with markup similar to the following.
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "alternates": [
+            ...
+            {
+                ...
+                "hide": {
+                    "storeOfficeAddin": {
+                        "officeAddinId": "b5a2794d-4aa5-4023-a84b-c60a3cbd33d4",
+                        "assetId": "WA999999999"
+                    }
+                },
+                "requirements": {
+                    "capabilities": [
+                        {
+                            "name": "MailBox",
+                            "minVersion": "1.10"
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.autoRunEvents.requirements
+
+The "extensions.autoRunEvents" property configures an add-in to run specified code automatically in response to specified events. The "requirements" subproperty can be used to block this behavior in some versions of Office.
+
+For example, suppose an Outlook add-in is configured to autolaunch in response to the **OnMailSend** event and suppose that the code in the function that runs requires the **Mailbox 1.13** requirement set. But the add-in has other features that would be useful Office versions that only support version 1.12. To ensure that the add-in is installable on versions that support 1.12, a developer can set the “extensions.requirements.capabilities” property to the requirement set **MailBox 1.12** instead of 1.13. But to block the autolaunch feature in versions that don't support 1.13, the developer can add an “extensions.autoRunEvents.requirements.capabilities” property that specifies **Mailbox 1.13**. The following is an example.
```

Comment:
```suggestion
For example, suppose an Outlook add-in is configured to autolaunch in response to the **OnMailSend** event and suppose that the code in the function that runs requires the **Mailbox 1.13** requirement set. But the add-in has other features that would be useful in Office versions that only support version 1.12. To ensure that the add-in is installable in versions that support 1.12, a developer can set the "extensions.requirements.capabilities" property to the requirement set **MailBox 1.12** instead of 1.13. But to block the autolaunch feature in versions that don't support 1.13, the developer can add an "extensions.autoRunEvents.requirements.capabilities" property that specifies **Mailbox 1.13**. The following is an example.
```